src/lib/checkbox/checkbox.html

```diff
@@ -1,15 +1,22 @@
 <template>
-  <div class="forge-checkbox-wrapper forge-form-field" part="root">
-    <div class="mdc-checkbox" part="container">
-      <slot name="input"></slot>
-      <div class="mdc-checkbox__background" part="background">
-        <svg class="mdc-checkbox__checkmark" viewBox="0 0 24 24" part="checkmark-svg">
-          <path class="mdc-checkbox__checkmark-path" fill="none" d="M1.73,12.91 8.1,19.28 22.79,4.59" part="checkmark-path"></path>
+  <div class="forge-checkbox" part="checkbox">
+    <div id="container" class="container" part="input-container">
+      <slot name="input">
+        <input id="input" class="input" type="checkbox" part="input" />
+      </slot>
+      <div class="background" part="background">
+        <svg class="icon icon--checked" aria-hidden="true" viewBox="0 0 24 24" part="checkmark">
+          <path class="path path--checkmark" d="M1.73,12.91 8.1,19.28 22.79,4.59" part="checkmark-path"></path>
         </svg>
-        <div class="mdc-checkbox__mixedmark" part="mixedmark"></div>
+        <svg class="icon icon--indeterminate" aria-hidden="true" viewBox="0 0 24 24" part="mixedmark">
+          <line x1="2" y1="12" x2="22" y2="12" part="mixedmark-path"></line>
+        </svg>
+        <forge-focus-indicator target="container" part="focus-indicator"></forge-focus-indicator>
```

Comment:
For the `<forge-focus-indicator>` and `<forge-state-layer>` elements we should use `exportparts`. See the slider and tab for an example.